examples/single-consumer/main.go

```diff
@@ -7,14 +7,25 @@ import (
 	"time"
 )
 
+func SampleHeaderFilterFn(headers []kafka.Header) bool {
```

Comment:
You need to move this function to the examples/single-consumer-with-header-filter-function :D